homeassistant/components/gdacs/__init__.py

```diff
@@ -194,7 +194,11 @@ def status_info(self):
     async def _generate_entity(self, external_id):
         """Generate new entity."""
         async_dispatcher_send(
-            self._hass, self.async_event_new_entity(), self, external_id
+            self._hass,
+            self.async_event_new_entity(),
+            self,
+            self._config_entry.title,
```

Comment:
If we set a unique id on the config entry we could use that here instead of title.

https://developers.home-assistant.io/docs/config_entries_config_flow_handler#unique-ids

Using the title as unique_id, although it technically works now, isn't optimal since we don't normally require the title to be stable. It could be overlooked if we change the title in the future.

Reply:
OK. That is indeed a cleaner approach.